### PR TITLE
add ParticleEffects

### PR DESCRIPTION
**Pull request by spirit126**

### CODE REVIEW

File: Assets/Scripts/Effects/SpawnParticleEffect.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            var locationsVector = transform.localPosition;
+            effectObject.name = $"ParticleEffectX{locationsVector.x}Y{locationsVector.y}";
+            effectObject.transform.localPosition = locationsVector;
+            if (!spawnParticleEffect.CompareTag("Circle"))
+            { 
+                effectObject.transform.rotation = transform.rotation;  
+            }
+        }
+    }
+}
```
Comment (SomethingWittyWithATechAspect) [inline Assets/Scripts/Effects/SpawnParticleEffect.cs:55]: eof nl

File: Assets/Scripts/Effects/SpawnParticleEffect.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        private void SpawnEffect()
+        {
+            var effectObject = Instantiate(spawnParticleEffect, GameObject.Find("Map").transform);
+            var locationsVector = transform.localPosition;
+            effectObject.name = $"ParticleEffectX{locationsVector.x}Y{locationsVector.y}";
+            effectObject.transform.localPosition = locationsVector;
+            if (!spawnParticleEffect.CompareTag("Circle"))
+            { 
+                effectObject.transform.rotation = transform.rotation;  
+            }
```
Comment (SomethingWittyWithATechAspect) [inline Assets/Scripts/Effects/SpawnParticleEffect.cs:52]: why not move to SpawnObject class as static method, pref with overloading existing function?

**Review by SomethingWittyWithATechAspect — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Effects/SpawnParticleEffect.cs (hunk 2 above)
Comment (spirit126) [reply Assets/Scripts/Effects/SpawnParticleEffect.cs:52]: done

File: Assets/Scripts/Effects/SpawnParticleEffect.cs (hunk 1 above)
Comment (spirit126) [reply Assets/Scripts/Effects/SpawnParticleEffect.cs]: done

**Review by SomethingWittyWithATechAspect — APPROVED** [review]

**Pull request merged**